### ISSUE TITLE
Cleaner UI for choosing resolution action

### ISSUE THREAD

**Issue by ChemiKhazi** [googlesamples/unity-jar-resolver#54]
When there are many jars being resolved, the popups that ask users to choose to keep or to use the latest version gets overwhelming.

A better UI might be one that list all the libraries being resolved and lets the user decide on what to do. 

![image](https://cloud.githubusercontent.com/assets/2573643/26235431/addfda66-3c9d-11e7-9238-33c073e2696c.png)

**Comment by Deepscorn** [comment]
By the way it may be useful to show all dependencies in that window. Hidden by default. For example, when choosing between support-...-24.0.0 and support-...-25.0.0 I need to see all "support-..." dependencies to make version aligning between them. And additionally, compare them with a compile sdk version of the project to decide

**Comment by stewartmiles** [comment]
@ChemiKhazi we no longer pop up UI when performing resolution if there are conflicts and instead try to resolve conflicts transparently and report the changes to the user after it's complete.  I could imagine a UI for setting up and viewing dependencies in a project though.  Now we have support for specifying dependencies via XML it would need to read / write our XML dependency files and that's about it.  If anyone's game for a stab at this we're happy to accept a pull request.

**Comment by stewartmiles** [comment]
We don't prompt for resolution actions w.r.t overwriting etc. anymore.  Resolution occurs in a single step removing all previously resolved files and replacing them with the results of the most recent resolution.

**Comment by Deepscorn** [comment]
@stewartmiles, I don't understand how can you completely remove user interaction, because, for example we need to use support-...-24.0.0 with android compileSdk=24 and support-...-25.0.0 with compileSdk=25. Also, some android libs need version align between them. Normally, in pure android development, this is done manually in build.gradle

**Comment by stewartmiles** [comment]
@Deepscorn , since the Gradle resolver was added and the old resolver was ripped out no user interaction has been required.  The new Gradle resolver will attempt to loosen package version expressions until it finds a compatible set of libraries.  If some libraries are specified twice it will choose the most recent version.  Each time it makes a change it reports a warning in the Unity console.  If the developer wants to customize the setup of Android libraries they want to use they can do so via the `*Dependencies.xml` files in their project which is now the preferred method of managing dependencies.

I could imagine a scenario where we provided UI for configuring dependencies (i.e editing the XML files) but that sounds like a different feature request to this.
